examples/reports_api_example.py

```diff
@@ -0,0 +1,23 @@
+''' Reports API client example usage. '''
+
+from livechat.reports.client import ReportsApi
+
+# Get number of chats occured during specified period.
+reports_api = ReportsApi.get_client(token='<your access token>')
+chats_occured = reports_api.total_chats(to='2020-09-14T23:59:59+02:00',
+                                        from_param='2020-09-01T00:00:00+02:00')
+print(chats_occured.text)
```

Comment:
Are those prints intentional?

Reply:
were suppose to be .json() but overall the prints were intentional.